### DISCUSSION TITLE
Minor Misspelling - VolumeToMesh.h

### DISCUSSION THREAD

**Discussion by jabbathenut** [AcademySoftwareFoundation/openvdb#1596] [General]
Hi,

I was combing through the VoulumeToMesh.h code, today.  On line 4786, I noticed a misspelling in an inline function name.  I thought I would point it out in case it matters to anyone.  I have included a snapshot of that section of the code.
<img width="1011" alt="VolumeToMesh_h_snippet png" src="https://user-images.githubusercontent.com/3082713/224376149-18545325-c718-4c5c-a1fa-b1d49c87f420.png">

**Comment by fdarling** [comment]
I submitted a bug report before I realized this discussion post about it. Here is the issue:

https://github.com/AcademySoftwareFoundation/openvdb/issues/2147
